lr-cli/cortx_setup/commands/resource/discover.py

```diff
@@ -15,13 +15,47 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-
+import time
 from ..command import Command
+from provisioner.commands import PillarSet
+from cortx.utils.discovery import Discovery
 
 
 class ResourceDiscover(Command):
+
+    def get_resource_map(self, resource_type='nodes[0]'):
+        retries = 30
+        status = ""
+        self.logger.debug("Running resource discover command")
+        # Currently since this is a bug once fixed
```

Comment:
Please mention bug-id on which we are blocked

Reply:
Done